### PR TITLE
Sw 6550 fix time-date bug by setting local timezone and using time.local

### PR DESCRIPTION
**Pull request by kate-49**

### CODE REVIEW

File: internal/sirius/format_date_and_time.go [hunk 1]
```diff
@@ -10,6 +10,10 @@ func FormatDateAndTime(formatForDateTime string, dateString string, displayLayou
 		return dateString
 	}
 	stringToDateTime, _ := time.Parse(formatForDateTime, dateString)
-	dateTime := stringToDateTime.Format(displayLayoutDateTime)
+	dateTime := stringToDateTime.Local().Format(displayLayoutDateTime)
 	return dateTime
 }
+
+func isDST() bool {
```
Comment (denialanderror) [inline internal/sirius/format_date_and_time.go:17]: What does DST stand for?

Comment (denialanderror) [reply internal/sirius/format_date_and_time.go:17]: Also, this function is only used in tests

**Review by denialanderror — CHANGES_REQUESTED** [review]

File: internal/sirius/format_date_and_time.go (hunk 1 above)
Comment (kate-49) [reply internal/sirius/format_date_and_time.go:17]: Daylight saving time, do you want it moved into a test suite instead?

Comment (denialanderror) [reply internal/sirius/format_date_and_time.go]: If it is only used for tests, it should be in the test suite. Is it needed though? It looks like you are only using it to conditionally re-format dates in tests. Can't you instead always re-format the dates in the tests and not bother with the check?

Comment (kate-49) [reply internal/sirius/format_date_and_time.go]: I only want to reformat dates half of the year when time.Local (i.e. British Summer Time) doesn't line up with Time.Now (which always uses UTC and is the default in Go). We get a UTC date from Sirius so I do need this reformatting.

File: internal/sirius/get_deputy_events_test.go [hunk 2]
```diff
@@ -2,55 +2,62 @@ package sirius
...
 	"net/http/httptest"
 	"testing"
-
-	"github.com/ministryofjustice/opg-sirius-supervision-deputy-hub/internal/mocks"
-	"github.com/stretchr/testify/assert"
+	"time"
 )
 
+func AmendDateForDST(date string) string {
+	if time.Now().Local() != time.Now() {
```
Comment (denialanderror) [inline internal/sirius/get_deputy_events_test.go:15]: Do you need this condition? Can't you just apply `FormatDateAndTime` to all dates in the test and the formatter will handle whether it needs to be amended or not?

**Pull request merged**